src/modules/keyboardmanager/KeyboardManagerEngineLibrary/KeyboardEventHandlers.cpp

```diff
@@ -230,9 +289,55 @@ namespace KeyboardEventHandlers
                         it->second.winKeyInvoked = ModifierKey::Left;
                     }
 
-                    if (remapToShortcut)
+                    if (isRunProgram)
                     {
-                        // Get the common keys between the two shortcuts
+                        CreateOrShowProcessForShortcut(std::get<Shortcut>(it->second.targetShortcut));
```

Comment:
Maybe this needs to be on a different thread.

Reply:
Sounds good, I love it more.